pkg/podexecutor/staticpod.go

```diff
@@ -161,6 +164,17 @@ func (s *StaticPod) CurrentETCDOptions() (opts executor.InitialOptions, err erro
 	return
 }
 
+const (
+	basePath    = "/var/lib/rancher/rke2"
```

Comment:
I dont think this should be hard coded to /var/lib/rancher/rke2, the datadir for rke2 can be changed with an option

Reply:
Good point. I'll look to see how this can be avoided.

Reply:
Resolved. Thanks!